src/smart/SmartPatient.js

```diff
@@ -15,24 +15,13 @@ export function SmartPatient() {
     isImmunosuppressed: false,
     isPregnant: false,
     isPregnantConcerned: false,
-    isSymptomatic: false
+    isSymptomatic: false,
+    isToggleChanged: false
   });
 
-  const { output: dashboardInput, isLoadingCdsData: isLoadingCdsData } = useCds(patientData, toggleStatus, isToggleChanged);
+  const { output: dashboardInput, isLoadingCdsData: isLoadingCdsData } = useCds(patientData, toggleStatus);
```

Comment:
eslint is now throwing the following warning for the `isLoadingCdsData` declaration here:

> Destructuring assignment isLoadingCdsData unnecessarily renamed.

I'm not sure why this wasn't being recognized before, but should we consider switching to:
```javascript
  const { output: dashboardInput, isLoadingCdsData } = useCds(patientData, toggleStatus);

```